composer.json

```diff
@@ -18,22 +18,21 @@
     "require": {
         "composer/installers": "^2.0",
         "drupal/admin_toolbar": "^3",
-        "drupal/config_ignore": "2.x-dev",
-        "drupal/config_split": "^1",
-        "drupal/core-composer-scaffold": "^10",
-        "drupal/core-recommended": "^10",
+        "drupal/config_ignore": "^3",
+        "drupal/config_split": "^2",
+        "drupal/core-composer-scaffold": "^11",
+        "drupal/core-recommended": "^11",
         "drupal/devel": "^5",
-        "drupal/workbench": "^1",
-        "drupal/workbench_tabs": "^1",
-        "drush/drush": ">=11"
+        "drupal/workbench": "^1.5",
+        "drupal/workbench_tabs": "^1.8",
+        "drush/drush": "^13"
     },
     "require-dev": {
         "behat/behat": "^3.12",
-        "behat/mink-goutte-driver": "^2.0",
```

Comment:
It sounds like we don't need this anymore. Is this correct?
https://packagist.org/packages/behat/mink-goutte-driver

Drupal already has https://packagist.org/packages/behat/mink-browserkit-driver as a requirement.

Reply:
Hmm. Maybe there's more to do here:

> In GoutteFactory.php line 66:
> 
>  Install MinkGoutteDriver in order to use goutte driver.
https://circleci.com/gh/palantirnet/drupal-skeleton/708

Reply:
Fixed by updating https://github.com/palantirnet/the-build/pull/237/commits/02cd8ee0495ace361ace8ccee0af697a74b14b33